src/analysis.ts

```diff
@@ -100,8 +100,9 @@ export function utonalFundamental(frequencies: number[], maxDivisor = 23) {
   return divisor * frequencies[0]
 }
 
-// Interval matrix a.k.a the modes of a scale
-export function intervalMatrix(scale: Scale) {
+// Interval matrix a.k.a the modes of a scale.
+export function intervalMatrix(scale: Scale): [[Interval]] {
+  console.log('intervalMatrix was called')
```

Comment:
No `console.log` in PRs without marking them as drafts.

Reply:
Sorry, I forgot to check.